ext/civicrm_admin_ui/managed/SavedSearch_Administer_Profiles.mgd.php

```diff
@@ -225,6 +225,17 @@
                   'join' => '',
                   'target' => '_blank',
                 ],
+                [
+                  'path' => 'civicrm/admin/uf/group/copy?action=copy&gid=[id]',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Copy'),
+                  'style' => 'default',
+                  'condition' => [],
+                  'entity' => '',
+                  'action' => '',
```

Comment:
Instead of hard-coding the URL in a search display, add `<copy>` to the `<paths>` in `UFGroup.xml` and then you can just do this:

```suggestion
                  'path' => '',
                  'icon' => 'fa-external-link',
                  'text' => E::ts('Copy'),
                  'style' => 'default',
                  'condition' => [],
                  'entity' => 'UFGroup',
                  'action' => 'copy',
```